package.json

```diff
@@ -11,24 +11,22 @@
     "url": "https://github.com/mobify/nightwatch-commands/issues"
   },
   "devDependencies": {
-    "nightwatch": "0.9.1",
+    "nightwatch": "0.9.8",
     "nodeunit": "~0.8.4",
     "grunt": "0.4.5",
     "grunt-cli": "~0.1.13",
     "grunt-eslint": "^17.0.0",
     "mobify-code-style": "^2.3.6"
   },
   "scripts": {
-    "install": "node selenium/installation/install.js",
+    "install": "node selenium/install.js",
     "test": "./node_modules/.bin/grunt lint; ./node_modules/.bin/grunt test"
   },
   "homepage": "https://github.com/mobify/nightwatch-commands",
   "dependencies": {
-    "mkdirp": "^0.5.1",
     "async": "^0.2.10",
     "chalk": "^0.4.0",
-    "mkdirp": "^0.5.0",
-    "unzip": "^0.1.9",
-    "rimraf": "^2.2.8"
+    "mkdirp": "^0.5.1",
```

Comment:
It would be a nice-to-have to get _all_ dependency versions pinned down with this update.